tests/unittest_mlagent/unittest_mlagent.cc

```diff
@@ -116,14 +117,15 @@ TEST (testMLAgent, GetModelInvalidModel_n)
   g_value_init (&val, G_TYPE_STRING);
 
   g_autofree gchar *uri
-      = g_strdup_printf (valid_uri_format_literal, "InvalidModelName", UINT32_MAX);
+      = g_strdup_printf (invalid_uri_format_literal, "InvalidModelName", UINT32_MAX);
```

Comment:
This is the case for the valid URI format with an invalid model name (which is an invalid URI).

Reply:
I see. I updated the code ml_agent.c to handle fallback case properly, PTAL